Openshift-EE/pipelines/GitLab/stages/4-chaos/XMSV-gitlab-unicorn-failure/gitlab-unicorn-kill

```diff
@@ -1,5 +1,72 @@
 #!/bin/bash
 set -x
 
-echo "***Killing the gitlab unicorn Pod***"
-sleep 2
+pod() {
+sshpass -p $pass_e2e ssh -o StrictHostKeyChecking=no $user_e2e@$ip -p $port 'cd e2e-openshift && bash Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/1PP5-app-kill/app-kill node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"'
```

Comment:
@shashank855 test script path should be `Openshift-EE/pipelines/GitLab/stages/4-chaos/XMSV-gitlab-unicorn-failure/gitlab-unicorn-kill`